From fb37ed1bee16041e3f3f3c598bb1a2ab4cfe582e Mon Sep 17 00:00:00 2001
From: Avinash M N <avimn@amd.com>
Date: Thu, 22 Jun 2017 13:06:31 +0530
Subject: [PATCH 2058/2094] drm/amd/amdgpu: get maximum and used UVD handles
 (v4)

return -ENODATA for asics with unlimited sessions. This change is an
extension of commit 0f91b2 as per amd-staging-4.11

Signed-off-by: Avinash M N <avimn@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c
index e7554cb..c14a61e 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_kms.c
@@ -694,7 +694,7 @@ static int amdgpu_info_ioctl(struct drm_device *dev, void *data, struct drm_file
 				return copy_to_user(out, &handle,
 					min((size_t)size, sizeof(handle))) ? -EFAULT : 0;
 			} else {
-				return -EINVAL;
+				return -ENODATA;
 			}
 
 			break;
-- 
2.7.4

